homeassistant/components/firmata/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+  "domain": "firmata",
+  "name": "Firmata",
+  "documentation": "https://www.home-assistant.io/components/firmata",
```

Comment:
Activate config flow here. See other integrations with config flow for example.

Reply:
So I guess I don't want to activate the config flow?